doctr/io/pdf.py

```diff
@@ -24,15 +25,17 @@ def read_pdf(file: AbstractFile, scale: float = 2, **kwargs: Any) -> List[np.nda
     Args:
         file: the path to the PDF file
         scale: rendering scale (1 corresponds to 72dpi)
+        kwargs: additional parameters to :func:`pypdfium2.render_pdf_base`
     Returns:
-        the list of pages decoded as numpy ndarray of shape H x W x 3
+        the list of pages decoded as numpy ndarray of shape H x W x C
     """
 
-    if not isinstance(file, (str, Path, bytes)):
+    if isinstance(file, Path):
+        file = str(file)
+    if not isinstance(file, (str, bytes)):
         raise TypeError("unsupported object type for argument 'file'")
 
-    if isinstance(file, (str, Path)) and not Path(file).is_file():
+    if isinstance(file, str) and not os.path.isfile(file):
         raise FileNotFoundError(f"unable to access {file}")
 
-    # Read pages with fitz and convert them to numpy ndarrays
-    return [np.asarray(img) for img, _ in pdfium.render_pdf(file, scale=scale)]
+    return [np.asarray(img) for img, _num in pdfium.render_pdf_topil(file, scale=scale, **kwargs)]
```

Comment:
While the 1.0.0 release is in beta, perhaps we could constrain the version specifier and keeping the old behavior?

Additionally, `_num` isn't being used, would you mind reverting back to `_` (it's for the lint checker)

Reply:
Okay, will change it too.
(I personally tend not to use `_` for unused variables since it can conflict with the `gettext.gettext()`, which is commonly aliased to an underscore, but you're right that's not relevant here.)